src/app_service/service/community/service.nim

```diff
@@ -1056,14 +1098,15 @@ QtObject:
         return
 
       self.events.emit(SIGNAL_COMMUNITY_MEMBER_APPROVED, CommunityMemberArgs(communityId: communityId, pubKey: newMemberPubkey))
+      self.events.emit(SIGNAL_COMMUNITY_EDITED, CommunityArgs(community: self.joinedCommunities[communityId]))
```

Comment:
Not sure if it belongs here, there is a more specific signal one line above. Shouldn't we update the places which do not react to `SIGNAL_COMMUNITY_MEMBER_APPROVED` instead of emitting `SIGNAL_COMMUNITY_EDITED` ?

Reply:
Agree, it was not needed.